src/ansible_navigator/actions/settings.py

```diff
@@ -0,0 +1,184 @@
+"""The ``settings`` subcommand action."""
+
+from dataclasses import asdict
+from typing import Tuple
+
+from .._yaml import human_dump
+from ..action_base import ActionBase
+from ..action_defs import RunStdoutReturn
+from ..app_public import AppPublic
+from ..configuration_subsystem import PresentableSettingsEntries
+from ..configuration_subsystem import PresentableSettingsEntry
+from ..configuration_subsystem import to_presentable
+from ..steps import TypedStep
+from ..ui_framework import Color
+from ..ui_framework import CursesLinePart
+from ..ui_framework import CursesLines
+from ..ui_framework import Decoration
+from ..ui_framework import Interaction
+from . import _actions as actions
+from . import run_action
+
+
+def color_menu(colno: int, colname: str, entry: PresentableSettingsEntry) -> Tuple[int, int]:
+    # pylint: disable=unused-argument
+    """Color the menu.
+
+    :param colno: Column number
+    :param colname: Column name
+    :param entry: Column value
+    :returns: Constants that curses uses to color a line of text
+    """
+    if entry.default:
+        return Color.GREEN, Color.BLACK
+    return Color.YELLOW, Color.BLACK
+
+
+CONTENT_HEADING_DEFAULT = "{name} (current value/default value: {current_value})"
+CONTENT_HEADING_NOT_DEFAULT = (
+    "{name} (current value: {current_value})  (default value: {default_value})"
+)
+
+
+def content_heading(obj: PresentableSettingsEntry, screen_w: int) -> CursesLines:
+    """Create a heading for the setting entry showing.
+
+    :param obj: The content going to be shown
+    :param screen_w: The current screen width
+    :returns: The heading
+    """
+    if obj.default:
+        text = CONTENT_HEADING_DEFAULT.format(**asdict(obj))
+        color = Color.GREEN
+    else:
+        text = CONTENT_HEADING_NOT_DEFAULT.format(**asdict(obj))
+        color = Color.YELLOW
+
+    fill_characters = screen_w - len(text) + 1
+    heading_line = f"{text}{' ' * fill_characters}"
```

Comment:
Did you mean
```suggestion
    heading_line = text.ljust(screen_w + 1)
```
?

Reply:
_(also, what is that `+ 1`?)_

Reply:
yes :) ljust is better here.......

The +1 is because screen width was poorly named long time ago:

```
@property
    def _screen_width(self) -> int:
        """return the screen width

        :return: the current screen width
        """
        return self._screen.getmaxyx()[1]
```
which returns the max x and y, not really a width and height, this could be handled better with a property, but this would be a big change...